test/e2e/actions/actions.go

```diff
@@ -34,7 +34,7 @@ func UpdateCluster(newData *model.TestDataProvider) {
 	})
 
 	By("Check attributes\n", func() {
-		uCluster := mongocli.GetClustersInfo(newData.Resources.ProjectID, newData.Resources.Clusters[0].Spec.ClusterSpec.Name)
+		uCluster := mongocli.GetClustersInfo(newData.Resources.ProjectID, newData.Resources.Clusters[0].Spec.GetClusterName())
```

Comment:
in all places we use ClusterSpec.Name, we use the new access method which checks for Advanced/Regular names